provider/datagen/tests/data/segmenter/Thai_codepoints_exclusive_model4_heavy/weights.json

```diff
@@ -1 +1 @@
.04586194455623627, 0.07499399036169052, -0.0027261050418019295, -0.91718989610672, 0.3603958189487457, 1.1646023988723755, -2.158501625061035, 0.20802681148052216, -0.18884986639022827, 0.1469474732875824, -0.09517357498407364, -8.43395709991455, 0.5940433740615845, -0.021385658532381058, -5.005634307861328, 0.16691741347312927, 1.641148567199707, 0.5658538937568665, -0.09278318285942078, -3.509357452392578, 0.0972050204873085, 0.3486752212047577, -0.17438213527202606, -0.19962726533412933, 0.2022196352481842, -0.21797801554203033, -0.1684822142124176, 0.19542740285396576, -0.2638742923736572, 0.45146629214286804, -0.06722266227006912, -3.0619499683380127, 0.1935032457113266, -1.3368524312973022, -0.4287770390510559, -0.7205678820610046, -1.7994238138198853, 0.05637070909142494, -0.01789172925055027, 0.32718411087989807, 0.17517495155334473, -0.18564128875732422, -0.07910347729921341, 0.38142910599708557, -2.0775146484375, 0.3218713402748108, -1.9596542119979858, 0.03945983201265335, -1.4417067766189575, 0.142643004655838, 1.2647262811660767, -1.4738622903823853, 0.06818267703056335, 0.30327489972114563, -0.23153169453144073, 0.027902433648705482, -0.025882095098495483, -0.18646584451198578, -0.28470170497894287, -0.43705058097839355, -0.844118058681488, -0.9221940040588379, -0.5004085898399353, -0.5048850774765015, 0.4583062529563904, -0.3922697603702545, -0.15811210870742798, -0.23841692507266998, -0.4636690020561218, 1.4435592889785767, -1.3974899053573608, -1.002795934677124, 0.29495131969451904, 0.24911780655384064, 0.004633332137018442, -0.04015856981277466, 0.07644599676132202, 0.07159379869699478, -0.07893693447113037, 0.17210043966770172, 0.4089183807373047, -0.026284806430339813, -0.0038357852026820183, 0.05243982374668121, -0.04277276247739792, 0.808382511138916, -0.4768460988998413, -2.6192848682403564, -6.690356731414795, 0.041267890483140945, -0.3985750377178192, -0.9322471618652344, -0.087257519364357, -0.42741361260414124, -0.11142417043447495, 0.3644208610057831, 0.17999224364757538, -0.43283113837242126, -0.28162455558776855, -0.31483158469200134, -0.30528444051742554, 0.9852316975593567, -0.09494172781705856, 0.2711966037750244, -0.047579120844602585, -0.20159940421581268, 0.0625423938035965, -0.2600880563259125, 0.011297800578176975, 0.16690939664840698, -1.3797320127487183, -0.06731226295232773, 1.745949149131775, -1.2130578756332397, -3.363910675048828, -0.8687061071395874, 0.4875473380088806, -0.07926753908395767, 0.098166823387146, -0.835883617401123, 0.34717410802841187, 0.005618626717478037, -0.8355814218521118, 0.12678614258766174, 0.013020575046539307, 0.8142793774604797, 0.16666975617408752, 0.11141733825206757, 0.5674348473548889, 0.021024350076913834, 0.17301759123802185, -0.2927340865135193, 0.3180336654186249, -0.2991063892841339, 0.10294848680496216, 0.09842441976070404, 0.10851484537124634, 0.10525725781917572, -0.12914223968982697, -1.1543477773666382, 0.06883743405342102, -0.16690213978290558, -1.755323529243
```

Comment:
Question/Issue: Why are there diffs in the source LSTM data files? Please avoid diffs in these files

Reply:
I don't think these are source data files

Reply:
oh they are